### PR TITLE
V2 coverage proposal payload

### PR DESCRIPTION
**Pull request by yonikesel**

### CODE REVIEW

File: src/payloads/V2CoveragePaymentPayload.sol [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    address public constant CHAOS_RECIPIENT = 0xbC540e0729B732fb14afA240aA5A047aE9ba7dF0;
+
+    address public constant AUSDC_TOKEN = 0xBcca60bB61934080951369a648Fb03DF4F96263C;
+    address public constant AAVE_TOKEN = 0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9;
+
+    // ~175,000 aUSDC = $175,000
+    // Small additional amount to handle remainder condition during streaming
+    // duration 150 days = 12960000 [seconds] --> (175000e6 + [12960000 - 3200000]) % 12960000 = 0
+    // https://github.com/bgd-labs/aave-ecosystem-reserve-v2/blob/release/final-proposal/src/AaveEcosystemReserveV2.sol#L229-L233
+    uint256 public constant AUSDC_STREAM_AMOUNT = 175000e6 + 11840000;
```
Comment (Maltmark) [inline src/payloads/V2CoveragePaymentPayload.sol:32]: @yonikesel can you explain the math/numbers

File: script/DeployProposalPayload.s.sol [hunk 2]
```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 
 import "@forge-std/console.sol";
 import {Script} from "@forge-std/Script.sol";
-import {ProposalPayload} from "../src/payloads/LTTailDecemberPayload.sol";
+import {ProposalPayload} from "../src/payloads/V2CoveragePaymentPayload.sol";
```
Comment (yhayun) [inline script/DeployProposalPayload.s.sol]: Let's not touch this file. It's more of a reference. 
see script/specific path - and add a file there for deployment

File: src/payloads/V2CoveragePaymentPayload.sol [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    // TODO: fix the amount of tokens according to aave price
+    // ~1358 aAAVE = $75,000
+    // TODO: what is the remainder here?
+    // Small additional amount to handle remainder condition during streaming
+    // duration 150 days = 12960000 [seconds] --> (1358e18 + [12960000 - 12320000]) % 12960000 = 0
+    // https://github.com/bgd-labs/aave-ecosystem-reserve-v2/blob/release/final-proposal/src/AaveEcosystemReserveV2.sol#L229-L233
+    uint256 public constant AAVE_STREAM_AMOUNT = 1358e18 + 640000;
```
Comment (yhayun) [inline src/payloads/V2CoveragePaymentPayload.sol]: We need to sign off on this number. Ori need to approve price

**Review by yhayun — APPROVED** [review]
@yonikesel  looks good!
few nits- good job oberall

File: script/DeployProposalPayload.s.sol (hunk 2 above)
Comment (yonikesel) [reply script/DeployProposalPayload.s.sol]: reverted and created a new file under script/specific

File: src/payloads/V2CoveragePaymentPayload.sol [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    address public constant AAVE_TOKEN = 0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9;
+
+    // ~175,000 aUSDC = $175,000
+    // Small additional amount to handle remainder condition during streaming
+    // duration 150 days = 12960000 [seconds] --> (175000e6 + [12960000 - 11840000]) % 12960000 = 0
+    // https://github.com/bgd-labs/aave-ecosystem-reserve-v2/blob/release/final-proposal/src/AaveEcosystemReserveV2.sol#L229-L233
+    uint256 public constant AUSDC_STREAM_AMOUNT = 175000e6 + 11840000;
+
+    // TODO: fix the amount of tokens according to aave price
+    // ~1242 aAAVE = $75,000
```
Comment (yhayun) [inline src/payloads/V2CoveragePaymentPayload.sol:35]: Let's add a comment:
Aave token 30 days TWAP price 22/11-21/12 is $60.378

**Review by yhayun — APPROVED** [review]

**Pull request merged**